libs/wingsdk/src/testing/simulator.ts

```diff
@@ -653,3 +734,12 @@ export interface ConnectionData {
   /** A name for the connection. */
   readonly name: string;
 }
+
+interface SimulatorServerRequest {
```

Comment:
Should we export this interface?

Reply:
Per the discussion in the other thread -- I think for now it's best to avoid adding this to the public API surface area if there isn't a need for it, so that folks don't assume our simulator's HTTP API is stable.